lib/Service/UserMigrationService.php

```diff
@@ -264,7 +221,7 @@ protected function importUser(IImportSource $importSource,
 
 		$user = $this->userManager->createUser($data['uid'], \OC::$server->getSecureRandom()->generate(10, ISecureRandom::CHAR_ALPHANUMERIC));
 
-		if ($user === false) {
+		if (!$user instanceof IUser) {
```

Comment:
Had to reread to grok this, `!($user instanceof IUser)` would be clearer

Reply:
See https://github.com/nextcloud/user_migration/pull/45